firestore/cloud-client/vector_search.py

```diff
@@ -48,7 +50,7 @@ def vector_search_basic(db):
 
 
 def vector_search_prefilter(db):
-    # [START firestore_vector_search_basic]
+    # [START firestore_vector_search_prefilter]
```

Comment:
Just want to double-check that you want to remove the old region tag. If it's being used in the docs somewhere, this could break the link. It looks like it's not used anywhere, but wanted to be sure

Reply:
This was added last week or so, and is actually an accidental duplicate of the above region tag.